app/controllers/api_controller/tags.rb

```diff
@@ -67,15 +75,18 @@ def edit_resource_tags(type, id, data)
       entry.tag
     end
 
+    def delete_resource_tags(_type, id, *)
+      entry = Classification.find_by_tag_id(id)
+      raise BadREquestError, "Failed to find tag resource" unless entry
+      entry.destroy
+    end
```

Comment:
we'll need to return action signature, similar to what is done in delete_resource_action(klass, type, id).